configure.ac: disable -Werror

Disable the mandatory flag -Werror in configure.ac.

Signed-off-by: Karoly Kasza <kaszak@gmail.com>

--- openvmtools-10.1.5/configure.ac.orig	2015-06-17 10:02:00.000000000 +0200
+++ openvmtools-10.1.5/configure.ac	2015-06-17 10:02:00.000000000 +0200
@@ -1218,7 +1218,7 @@
 
 ### General flags / actions
 CFLAGS="$CFLAGS -Wall"
-CFLAGS="$CFLAGS -Werror"
+# CFLAGS="$CFLAGS -Werror"
 
 # Suppress warnings from clang when taking the address of packed struct
 # member fields when compiling hgfsServer.c

